libpod/oci_conmon_common.go

```diff
@@ -937,7 +938,14 @@ func (r *ConmonOCIRuntime) ExitFilePath(ctr *Container) (string, error) {
 	if ctr == nil {
 		return "", fmt.Errorf("must provide a valid container to get exit file path: %w", define.ErrInvalidArg)
 	}
-	return filepath.Join(r.exitsDir, ctr.ID()), nil
+	return filepath.Join(r.persistDir, ctr.ID(), "exit"), nil
+}
+
+func (r *ConmonOCIRuntime) OOMFilePath(ctr *Container) (string, error) {
+	if ctr == nil {
+		return "", fmt.Errorf("must provide a valid container to get oom file path: %w", define.ErrInvalidArg)
+	}
```

Comment:
what is the point of this check here? In the code paths were this is called ctr can never be nil or it would have long before panicked already.

Reply:
That is true. The ExitFilePath function does this check so followed the semantics there for oom file.